vars/detectDockerAgent.groovy

```diff
@@ -4,9 +4,18 @@ Map call(Map args = [:]) {
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
     dockerImage = inputManifest.ci?.image?.name ?: 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
     dockerArgs = inputManifest.ci?.image?.args
+    // Using default javaVersion as jdk-17
+    String javaVersion = 'jdk-17'
+    java.util.regex.Matcher match = (dockerArgs =~ /jdk-\d+/) 
+    if (match.find()) {
+            def line = match[0]
+           javaVersion = line
```

Comment:
Spacing issues.

Reply:
Can you rename `line` to a better var name.

Reply:
Fixed Peter